redisinsight/ui/src/pages/browser/modules/key-details/components/modules-type-details/ModulesTypeDetails.tsx

```diff
@@ -1,14 +1,22 @@
 import React from 'react'
-import { EuiFlexGroup, EuiFlexItem, EuiText, EuiTitle } from '@elastic/eui'
+import { EuiText, EuiTitle } from '@elastic/eui'
 import { useHistory } from 'react-router-dom'
 import { useSelector } from 'react-redux'
 
 import { Pages } from 'uiSrc/constants'
 import { connectedInstanceSelector } from 'uiSrc/slices/instances/instances'
 
-import styles from '../unsupported-type-details/styles.module.scss'
+import styles from './styles.module.scss'
+import TextDetailsWrapper from '../text-details-wrapper/TextDetailsWrapper'
 
-const ModulesTypeDetails = ({ moduleName = 'unsupported' }: { moduleName: string }) => {
+const ModulesTypeDetails = (
+  { moduleName = 'unsupported', 
+    onClose 
+  }: 
+    { moduleName: string, 
+      onClose: () => void,
+    }
+  ) => {
```

Comment:
for component props I usually prefer extracting their type in `{$componentName}Props` type such as:
```suggestion
type ModulesTypeDetailsProps = {
  moduleName: string, 
  onClose: () => void,
}
const ModulesTypeDetails = ({ moduleName = 'unsupported', onClose }: ModulesTypeDetailsProps) => {
```
that way I think it's easier to read, extend, export the type, etc